src/deepsparse/utils/state.py

```diff
@@ -45,14 +49,40 @@ def create_state(self, new_state: dict):
         self._current_state = new_state
 
 
-class InferenceState(State):
+class TimerState:
+    """TimerState shared among all InferenceState"""
+
+    _timer = None
```

Comment:
why is this static and not a class property?

Reply:
we don't want to share the timer between inference states. each new inference state (outside of copies ofc) should have their own timer so that the timer can hold the specific timings for a single inference